bungee/src/main/java/me/leoko/advancedban/bungee/utils/CloudNetCloudPermsOfflineUser.java

```diff
@@ -0,0 +1,27 @@
+package me.leoko.advancedban.bungee.utils;
+
+import me.leoko.advancedban.utils.Permissionable;
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.driver.permission.IPermissionUser;
+
+import java.util.List;
+import java.util.UUID;
+
+public class CloudNetCloudPermsOfflineUser implements Permissionable {
+    private IPermissionUser permissionUser;
+
+    public CloudNetCloudPermsOfflineUser(String name) {
+    	final List <IPermissionUser> users = CloudNetDriver.getInstance().getPermissionManagement().getUsers(name);
+    	
+    	if (!users.isEmpty()) {
+        	this.permissionUser = users.get(0);
+    	} else {
+    		this.permissionUser = null;
+    	}
```

Comment:
The spacing is broken again.

Also no need for the else statement. Object attributes are null by default

Reply:
oh yeah I thought it was final, mb